### DISCUSSION TITLE
carousel 3d nextjs tailwind

### DISCUSSION THREAD

**Discussion by muhammadanas70** [tailwindlabs/tailwindcss#12221] [Help]
Is that easy to make carousel 3d nextjs + tailwind ?

**Comment by alex-krasikau** [comment]
Hey @MuhammadAnasHack!

Tailwind CSS doesn't impose any limitations in these regards. You can check these examples to find a good starting point:
- https://www.youtube.com/watch?v=2kwsknfpJFo&ab_channel=Raizensoft
- https://www.youtube.com/watch?v=06C2ppvh8gA&ab_channel=Raizensoft
